dockerfiles/docker-compose.yml

```
@@ -35,17 +35,15 @@ services:
   wrangler:
     build:
       context: ${PWD}
-      dockerfile: ${PWD}/common/dockerfiles/Dockerfile.wrangler
+      dockerfile: ${PWD}/common/packages/addons-inject/Dockerfile
     volumes:
-      - ${PWD}/common/dockerfiles/force-readthedocs-addons.js:/usr/src/app/docker/force-readthedocs-addons.js
+      - ${PWD}/common/packages/addons-inject/:/usr/src/app/packages/addons-inject/
     networks:
       readthedocs:
+    working_dir: /usr/src/app/packages/addons-inject/
     command: [
       "wrangler",
```

We need this to be as follows to work properly when executing `inv docker.up`

```suggestion
      "node_modules/.bin/wrangler",
```